docs/source/ci-cd.md

```diff
@@ -187,6 +188,85 @@ pipelines:
 ```
 
 
+## Jenkins
+To set up `rover` for use with Jenkins, you'll first want to consider what type of Jenkins `agent` you'll be using in your pipelines. The below samples are showcasing a golang pipeline utilizing Docker, but can be extended to meet your specific needs. 
+
+### Pipelines using Distributed Builds via Node Agent
```

Comment:
```suggestion
### Distributed builds via the `node` agent
```

Reply:
We use sentence casing for header titles, feel free to tweak this back if I've omitted an important word or two with this edit